docs/ru/managing_emotions.md

```diff
@@ -0,0 +1,44 @@
+# Управление дроном силой мысли
+
+## Введение
+
+Нейронные сети и дроны достаточно быстро развиваются в мире технологий. В управлении квадрокоптером за основу была взята искусственная нейронная сеть, состоящая из искусственных нейронов (программируемая система, которая имитирует подобие биологических нейронов).
+
+Нейронная сеть разделяется на несколько слоев:
+
+1. входной слой;
+2. скрытый слой;
+3. выходной слой.
+
+Целью данного проекта было создание управления Клевер 4 с помощью нейрошлема от компании «NEUROBOTICS». Мы использовали NeuroPlay-8M с 8-ми канальной системой беспроводной регистрации ЭЭГ данных человека на (гелевых) электродах. Программное обеспечение имеет поддержку устройств серии Нейробелт с интерфейсом Bluetooth 4.0 (BLE).
+
+Видеодемонстрация:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/uLR5NNcekfA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Установка
+
+Для начала нужно установить необходимые программы:
+
+1. Cortex.
+2. BioEcho.
+
+## Использование
+
+Программное обеспечение Cortex: классифицирует сигнал мозга в эмоцию.
+
+<img src="../assets/Cortex1.jpg" width="450">
+
+<img src="../assets/Cortex 2.jpg" width="450">

```

Comment:
On a somewhat related note, please don't use capital letters in file names unless absolutely necessary and avoid using spaces altogether.